keymaps/pycom-sync.json

```diff
@@ -2,7 +2,7 @@
   "atom-workspace": {
     "ctrl-alt-s": "pymakr:upload",
     "ctrl-shift-s": "pymakr:uploadFile",
-    "ctrl-alt-t": "pymakr:toggleREPL",
+    "ctrl-alt-t": "pymakr:toggle REPL",
```

Comment:
how come that this change fixed that keyboard shortcut?

Reply:
The commands were declared without spaces (openfile, toggleREPL, uploadfile etc), being rendered by atom just like it is written. A solution for that is writing the command in camelCase so it will be properly rendered (Upload File, Open File...) but for the command toggleREPL, for some reason, it needs to have a space between the words. Apparently I changed all the toggleREPL occurrences but I forgot the main one.